Core/Src/nero.c

```diff
@@ -26,22 +32,6 @@ static void send_mode_status()
 
 void set_mph(int8_t new_mph)
 {
-	//printf("mph %d", new_mph);
 	mph = new_mph;
-}
-
-osThreadId_t nero_monitor_handle;
-const osThreadAttr_t nero_monitor_attributes = {
-	.name = "NeroMonitor",
-	.stack_size = 32 * 8,
-	.priority = (osPriority_t)osPriorityHigh2,
-};
-
-void vNeroMonitor(void *pv_params)
-{
-	for (;;) {
-		send_mode_status();
-
-		osDelay(NERO_DELAY_TIME);
-	}
+	send_nero_msg();
```

Comment:
still think this should be removed.  A message reacting based on another node is not how we do any other part of our system.

Reply:
Its either this or setting a flag and having a task run that sends the nero message with the new data.

Reply:
Well rn ir using the default thread so that's a way.